docs/source/AdministratorGuide/ServerInstallations/environment_variable_configuration.rst

```diff
@@ -50,6 +50,9 @@ DIRAC_M2CRYPTO_SSL_METHODS
 DIRAC_NO_CFG
   If set to anything, cfg files on the command line must be passed to the command using the --cfg option.
 
+DIRAC_ROOT_PATH
+  If set, overwrites the value of DIRAC.rootPath.
```

Comment:
I think it might be useful to spell out why here:
```suggestion
  If set, overwrites the value of DIRAC.rootPath.
  Useful for using a non-standard location for `etc/dirac.cfg`, `runit/`, `startup/`, etc.
```